README.md

```diff
@@ -53,7 +53,7 @@ Dependencies:
   `--enable-aptx` and/or `--enable-aptx-hd`)
 - [libldac](https://github.com/EHfive/ldacBT) (when LDAC encoding support is enabled with
   `--enable-ldac`)
-- [pandoc](https://pandoc.org/) (when man pages build is enabled with `--enable-manpages`)
+- [python3-docutils](https://docutils.sourceforge.io) (when man pages build is enabled with `--enable-manpages`)
```

Comment:
Please, change `python3-docutils` to `docutils`, because python3-docutils is a package name in e.g. ubuntu, not project name itself.

Also, please update links `./doc/bluealsa*.md`. to `*.rst`.